infrastructure/src/main/java/org/corfudb/infrastructure/CorfuServer.java

```diff
@@ -305,15 +296,14 @@ public static void main(String[] args) {
     public static <T extends ServerChannel>
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') T is not used in the method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2326)